### ISSUE TITLE
Not able to log training and validation loss to visualise in tensor-board as tfevents?

### ISSUE THREAD

**Issue by pratikchhapolika** [huggingface/transformers#15063]
I am downloading the model https://huggingface.co/microsoft/Multilingual-MiniLM-L12-H384/tree/main microsoft/Multilingual-MiniLM-L12-H384 and then using it.

Transformer Version: '4.11.3'

I have written the below code:

```
def compute_metrics(eval_pred):
    logits, labels = eval_pred
   

    predictions = np.argmax(logits, axis=-1)
    
    acc = np.sum(predictions == labels) / predictions.shape[0]
    
    return {"accuracy" : acc}
```

```
model = tr.BertForSequenceClassification.from_pretrained("/home/pc/minilm_model",num_labels=2)
model.to(device)

print("hello")

training_args = tr.TrainingArguments(
    output_dir='/home/pc/proj/results2',          # output directory
    num_train_epochs=10,              # total number of training epochs
    per_device_train_batch_size=16,  # batch size per device during training
    per_device_eval_batch_size=32,   # batch size for evaluation
    learning_rate=2e-5,
    warmup_steps=1000,                # number of warmup steps for learning rate scheduler
    weight_decay=0.01,               # strength of weight decay
    logging_dir='./logs',            # directory for storing logs
    logging_steps=1000,
    evaluation_strategy="epoch",
    save_strategy="no"
)


trainer = tr.Trainer(
    model=model,                         # the instantiated 🤗 Transformers model to be trained
    args=training_args,                  # training arguments, defined above
    train_dataset=train_data,         # training dataset
    eval_dataset=val_data,             # evaluation dataset
    compute_metrics=compute_metrics
)

```
Is there way to retrieve my ( **I  want to use tensor-board**) :

1. **Training loss** for every epoch
2. **Validation loss** for every epoch

I do not see anything in my log directory apart from model arguments which is empty?

**How can I save my Training and validation loss so that tensorboard events captures it.**

**Comment by sid8491** [comment]
I am not so sure, but try to run your code without `logging_steps=1000,`, it might solve your problem.
You are training your model for 10 epochs but you are telling it to log it at every 1000th epoch.
